int64 numKeypoints
string template_name

---
int64 result
superglue_ros/KeypointsDict keypoints_dict